app/services/notifications/new_badge_achievement/send.rb

```diff
@@ -0,0 +1,45 @@
+# send notifications about the new badge achievement
+module Notifications
```

Comment:
How do you feel about adding a basic spec to this object?
The functionality is only covered by a request spec currently. I think a basic test for the service object would be nice.

Reply:
Great idea, will do so

Reply:
Done